core/trino-main/src/test/java/io/trino/block/BlockAssertions.java

```diff
@@ -227,6 +229,16 @@ public static Block createRandomLongDecimalsBlock(int positionCount, float nullR
                 () -> String.valueOf(RANDOM.nextLong())));
     }
 
+    public static Block createRandomLongsBlock(int positionCount, int numberOfUniqueValues)
+    {
+        checkArgument(positionCount >= numberOfUniqueValues, "numberOfUniqueValues must be between 1 and positionCount: %s but was %s", positionCount, numberOfUniqueValues);
+        int[] randomPartitions = chooseRandomUnique(positionCount, numberOfUniqueValues).stream().mapToInt(Integer::intValue).toArray();
```

Comment:
1. move `.mapToInt` and `.toArray()` to separate lines
2. Rename `randomPartitions` to `uniqueValues`